pac4j-jwt/pom.xml

```diff
@@ -33,6 +33,10 @@
 			<artifactId>pac4j-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>commons-codec</groupId>
```

Comment:
I don't like adding new dependencies and some times ago, I learned that you can handle base64 encoding / decoding just with the JDK:

Example:
```
byte[] message = "hello world".getBytes("UTF-8");
String encoded = DatatypeConverter.printBase64Binary(message);
byte[] decoded = DatatypeConverter.parseBase64Binary(encoded);
```